### ISSUE TITLE
iOS build has bugs in shader code

### ISSUE THREAD

**Issue by antpaw** [Unity-Technologies/PostProcessing#727]
##### What happened?
If you dont select anything in "Color Grading -> Mode" or if you select HDR. you will be greeted with very heavy graphic bugs and this error log on every frame in Xcode. But this only affects older iOS devises. This is very confusing and very hard to debug, because there is not a single warning displayed, that it only works in linear colorspace. you have to enable "Mode" first to see this warning.

```
        Game[1227:140807] Execution of the command buffer was aborted due to an error during execution. Discarded (victim of GPU error/recovery) (IOAF code 5)
```

##### Post-processing stack version (v1, v2)?
v2

##### Unity version, operating system, target platform (standalone windows, mac, iOS, PS4...)?
2018.3.0f2, osx mojave, iOS, LWRP

**Comment by lifangjie** [comment]
meet the same problem on A9 devices

**Comment by yomanx** [comment]
any fix guys?

**Comment by mydudechris** [comment]
Yeah, hitting the same issue. App is fine without post-proc. Turn on pp, enable color grading, same error above occurs. Trying to use ACES in my app

**Comment by mydudechris** [comment]
Updating to the latest version of Unity, .6, seemed to fix this for me.
